### PR TITLE
[ICDS Dashboard] Removed ff for infantomter and stadiometer awc tabular report

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/1112385193248823/1153005958632441/f

##### SUMMARY
Removed ff for infantomter and stadiometer awc tabular report

##### FEATURE FLAG
No

### CODE REVIEW

**Review by Rohit25negi — APPROVED** [review]
Looks good. did you test this locally it is working fine after FF removal.?

File: custom/icds_reports/tests/agg_tests/test_export_data.py [hunk 1]
```diff
@@ -1106,330 +1106,6 @@ def test_awc_infrastructure_export(self):
...
-            ]
-        )
-
-    def test_awc_infrastructure_export_with_beta(self):
-        self.assertListEqual(
-            AWCInfrastructureExport(
-                config={
-                    'domain': 'icds-cas'
-                }, beta=True
             ).get_excel_data(location=''),
```
Comment (Rohit25negi) [inline custom/icds_reports/tests/agg_tests/test_export_data.py:1109]: why the location is ='' not 'b1'?

**Review by Rohit25negi — COMMENTED** [review]
Left one comments mostly to know the rationale

**Comment by challabeehyv** [comment]
[AWC_Infrastructure.xlsx](https://github.com/dimagi/commcare-hq/files/4551122/AWC_Infrastructure.xlsx)

file without feature flag

File: custom/icds_reports/tests/agg_tests/test_export_data.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/test_export_data.py]: Removed it to check the default case. But better to folow the old test. SO added it back

**Pull request merged**